### ISSUE TITLE
[QT] Can't launch internal cores

### ISSUE THREAD

**Issue by Tatsuya79** [libretro/RetroArch#8336]
## Description

From the Desktop Menu you can't run an internal core such as FFMPEG or Image Viewer.

### Expected behavior

From Desktop Menu file browser view, double clicking or using "run" in the core docklet for a video or image file should run the relevant core.

### Actual behavior

The core listing is displayed on run action without ffmpeg or imageviewer present.

### Steps to reproduce the bug

1. Make sure you don't have the old external cores for ffmpeg or imageviewer
2. Go into file browser view
3. try to launch a picture or video file

### Version/Commit
current nighlty

### Environment information
win7 x64 gcc 7.3

**Comment by orbea** [comment]
Did this ever work? How about the retropad core?

**Comment by Tatsuya79** [comment]
That never worked I think, same for retropad, I don't see a way to start it from QT.

**Comment by orbea** [comment]
@bparker06 Maybe you would like to look at this one when you have time?

**Comment by bparker06** [comment]
Unable to reproduce here on Linux or Windows.

**Comment by orbea** [comment]
@bparker06 I can reproduce it, attempting to load an jpg file results in a core selection screen where I can not select the correct core.

**Comment by bparker06** [comment]
Apologies, I missed the words "file browser", I was testing with the image playlist...yes I can reproduce the issue with the file browser.

**Comment by orbea** [comment]
Yes, I neglected to test the image playlist. I suppose the problem is that its not possible to select builtin cores, but if the core is already assigned with a playlist item it works out just fine.
